### DISCUSSION TITLE
How to prevent changing power capping on shared HW resources

### DISCUSSION THREAD

**Discussion by rootfs** [Climatik-Project/Climatik-Project#151] [Announcements]
Per 07/29 discussion, there is a risk of changing CPU/GPU power capping/frequency that could impact other pods that don't have power capping policies.

**Comment by linericyang** [comment]
Resolving this policy conflict than sharing the same HW resource could be complicated. 

However, in most cases, dedicated cores/NUMA nodes/sockets/CPUs are allocated to LLMs on CPU for performance reasons. In such scenarios, we can only set power capping to those LLM with exclusive HW, and tune the frequency of the corresponding cores without impacting other processes.
